graphql2/graphqlapp/contactmethod.go

```diff
@@ -72,6 +73,12 @@ func (q *Query) UserContactMethod(ctx context.Context, id string) (*contactmetho
 
 func (m *Mutation) CreateUserContactMethod(ctx context.Context, input graphql2.CreateUserContactMethodInput) (*contactmethod.ContactMethod, error) {
 	var cm *contactmethod.ContactMethod
+	cfg := config.FromContext(ctx)
+
+	if input.Type == contactmethod.TypeWebhook && !cfg.ValidWebhookURL(input.Value) {
+		return nil, validation.NewFieldError("value", "URL disallowed by administrator")
```

Comment:
nit: something like "blacklisted" might be more clear

```suggestion
		return nil, validation.NewFieldError("value", "URL blacklisted by an administrator")
```

Reply:
In this case, it would be "not whitelisted" as it doesn't currently support blacklisting -- I think that's why it was phrased as a neutral term

Reply:
Alternatives:
URL blocked by administrator?
URL excluded by administrator?

Reply:
Maybe just "not allowed"?